### PR TITLE
Add Q8 Spiders

### PR DESCRIPTION
**Pull request by Cj-Malone**
Rel https://github.com/alltheplaces/alltheplaces/pull/4848#issuecomment-1438804346

I don't like this brand, its sub brands or its APIs.

### CODE REVIEW

File: locations/spiders/okq8.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    name = "okq8"
+    start_urls = [
+        "https://www.okq8.se/-/Station/GetGlobalMapStations?appDataSource=9d780912-2801-4457-9376-16c48d02e688"
+    ]
+
+    BRANDS = {
+        "OKQ8": {"brand": "OKQ8", "brand_wikidata": "Q2789310"},
+        "Tanka": {"brand": "Tanka", "brand_wikidata": "Q10690640"},
+        "IDS": None,  # {"brand": "Ids"} # Some kind of private? HGV?
+        "Eon": None,  # {"brand": "Eon"} # ?
```
Comment (Romeo-PHILLIPS) [inline locations/spiders/okq8.py:17]: "Eon" seems to be this https://www.st1biogas.se/hitta-station

File: locations/spiders/q8_italia.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+# Also not just Q8Italia
+
+
+class Q8ItaliaSpider(Spider):
+    name = "q8_italia"
+    item_attributes = {"brand": "Q8", "brand_wikidata": "Q1634762"}
+    start_urls = ["https://www.q8.it/geolocalizzatore/pv/all"]
+
+    BRANDS = {
+        "TANGO": None,  # TangoSpider
```
Comment (Romeo-PHILLIPS) [inline locations/spiders/q8_italia.py:15]: Tango wikidata is Q2423920

File: locations/spiders/okq8.py (hunk 1 above)
Comment (Cj-Malone) [reply locations/spiders/okq8.py:17]: There was literally 1 Eon and 30 something IDS. I don't think they are big enough to worry about, and I'm throwing them away on line 40 so data consumers don't have to worry either.

File: locations/spiders/q8_italia.py (hunk 2 above)
Comment (Cj-Malone) [reply locations/spiders/q8_italia.py:15]: Yeah but it's duplicate POIs from `tango.py` so I'm throwing them away on line 36. That way data consumers shouldn't have to do any de duplication across ATP data.

**Pull request merged**